### PR TITLE
add save, formatted audio cues to editor accessibility help dialog

### PR DESCRIPTION
**Pull request by meganrogge**
fix #195716

### CODE REVIEW

File: src/vs/editor/common/standaloneStrings.ts [hunk 1]
```diff
@@ -25,6 +25,12 @@ export namespace AccessibilityHelpNLS {
 	export const tabFocusModeOffMsg = nls.localize("tabFocusModeOffMsg", "Pressing Tab in the current editor will insert the tab character. Toggle this behavior {0}.");
 	export const tabFocusModeOffMsgNoKb = nls.localize("tabFocusModeOffMsgNoKb", "Pressing Tab in the current editor will insert the tab character. The command {0} is currently not triggerable by a keybinding.");
 	export const showAccessibilityHelpAction = nls.localize("showAccessibilityHelpAction", "Show Accessibility Help");
+	export const saveAudioCueDisabled = nls.localize("saveAudioCueDisabled", "`audioCues.save` is disabled, so an alert will occur when a file is saved.");
```
Comment (rzhao271) [inline src/vs/editor/common/standaloneStrings.ts:28]: Just to check, what is an alert? Is it an in-product notification, or something like a Windows dialog (which makes its own sound)?

File: src/vs/workbench/contrib/accessibility/browser/accessibilityContributions.ts [hunk 2]
```diff
@@ -92,6 +97,33 @@ class EditorAccessibilityHelpProvider implements IAccessibleContentProvider {
 				content.push(AccessibilityHelpNLS.editableEditor);
 			}
 		}
+		const screenReaderOptimized = this._accessibilityService.isScreenReaderOptimized();
+		const saveAudioCue = this._configurationService.getValue(AudioCue.save.settingsKey);
```
Comment (rzhao271) [inline src/vs/workbench/contrib/accessibility/browser/accessibilityContributions.ts:101]: It looks like these two audio cue variables can be placed inside the if statement.

File: src/vs/editor/common/standaloneStrings.ts (hunk 1 above)
Comment (meganrogge) [reply src/vs/editor/common/standaloneStrings.ts:28]: It's an in product notification

**Comment by smoothzebra** [comment]
ในวันที่ พ. 18 ต.ค. 2023 07:58 น. Megan Rogge ***@***.***>
เขียนว่า:

**Review by rzhao271 — APPROVED** [review]

**Pull request merged**